doc/devbox_setup.md

```diff
@@ -322,4 +322,4 @@ This repository contains various C sample applications that illustrate how to us
 [clang]:https://clang.llvm.org/
 [simple_samples]: ../iothub_client/samples/
 [serializer_samples]: ../serializer/samples/
-[latest-release]:https://github.com/Azure/azure-iot-sdk-c/releases/latest
\ No newline at end of file
+[latest-release]:https://github.com/Azure/azure-iot-sdk-c/releases/latest
```

Comment:
Hi @tdevendra , this line seems to have an unnecessary change. Could you revert it please?

The other change in the date format is correct, thanks for the fix.